[docs] improve external plugin documentation (#159431)
elastic/kibana: docs/developer/plugin/plugin-tooling.asciidoc

You can leverage {kib-repo}blob/{branch}/packages/kbn-plugin-helpers[@kbn/plugin-helpers] to build a distributable archive for your plugin.
The package transpiles the plugin code, adds polyfills, and links necessary js modules in the runtime.

> You don't need to install the `plugin-helpers` dependency, as the `package.json` is already pre-configured if you created your plugin with `node scripts/generate_plugin` script.

Reviewer: Very minor nits: I'd break this into two sentences and change the word order. Your current changes are fine though.
```suggestion
You don't need to install the `plugin-helpers` dependency. If you created the plugin using `node scripts/generate_plugin` script, `package.json` is already pre-configured.
```
